src/style.css

```
@@ -0,0 +1,37 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

```suggestion
  background-image: url(./images/star.svg) no-repeat center;
```
це не помилка, але для скорочення коду можна три властивості разом прописувати
```